src/shared/sam/localLambdaRunner.ts

```diff
@@ -167,12 +188,20 @@ export async function invokeLambdaFunction(
             SAM_BUILD_MODE: 'debug',
         }
     }
-    await new SamCliBuildInvocation(samCliArgs).execute()
+
+    try {
+        await new SamCliBuildInvocation(samCliArgs).execute()
+    } catch (err) {
+        throw err as Error
```

Comment:
Does it not work to just omit the catch? Since this just rethrows.

Reply:
No, JS doesn't let you do `try...finally` without a catch, and the `finally` always should be triggered ASAP so the user doesn't have to see the temp build template for longer than necessary.

Reply:
Actually, hmm, TS doesn't throw a fit if I omit that...IIRC VS Code runs Node 10, which has no catch implemented according to that post. I'll remove it.